aunt-leahs-app/src/components/tab/tab.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+
+// USAGE:
+{/* <TabComponent
+        name = "Tab"
+        items={[
+            { title: 'First Tab', content: <h1>hello</h1>},
+            { title: 'Second Tab',content: <CustomButton size="small" color="primary">I am a button</CustomButton>},
+            { title: 'Third Tab', content: 'Some other class'}
+            ...
+        ]} /> */}
+
+function TabPanel(props) {
```

Comment:
Can you explain the purpose of this function and why it's defined outside of the class?

Reply:
Dissolved the classed and added it to the tab component class